docs/api/migration-guides/qiskit-algorithms-module.mdx

```diff
@@ -151,7 +151,7 @@ This guide describes these common configurations for algorithms that determine w
 from qiskit_aer.primitives import Sampler, Estimator
 
 sampler = Sampler(run_options={"approximation": True, "shots": None})
```

Comment:
@kevinsung the change is only for Estimator, not Sampler, correct? Should I leave this line as-is?

Reply:
As far as I know, "approximation" is not a valid run option, so I think it can be removed immediately from all examples of Sampler and Estimator. See https://github.com/Qiskit/qiskit-aer/issues/2083 though.

https://github.com/Qiskit/documentation/issues/319 Refers to passing `approximation=True` directly to the initializer of Estimator, as in those pages in the verify section.